sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/MetricsAdvisorLiveTestBase.cs

```diff
@@ -27,22 +27,22 @@ public MetricsAdvisorLiveTestBase(bool isAsync) : base(isAsync)
             Sanitizer = new MetricsAdvisorRecordedTestSanitizer();
         }
 
-        internal const string DetectionConfigurationId = "fb5a6ed6-2b9e-4b72-8b0c-0046ead1c15c";
-        internal const string IncidentId = "736eed64368bb6a372e855322a15a736-174e1756000";
-        internal const string AlertConfigurationId = "204a211a-c5f4-45f3-a30e-512fb25d1d2c";
-        internal const string AlertId = "17571a77000";
-        internal const string MetricId = "27e3015f-04fd-44ba-a20b-bc529a0aebae";
-        internal const string DataFeedId = "9860df01-e740-40ec-94a2-6351813552ba";
+        internal const string DetectionConfigurationId = "efaee305-f049-43ec-9f9b-76026d55c14a";
+        internal const string IncidentId = "aaa0ff1cfe41d89ed481f9ac19dbcd8e-17c76f2dc00";
+        internal const string AlertConfigurationId = "1c1575d8-b09e-40c3-a3c0-d459c64d8382";
+        internal const string AlertId = "17bbd8dec00";
+        internal const string MetricId = "390d1139-98fb-45af-b831-8d5ad61b150a";
+        internal const string DataFeedId = "151e5444-449f-441e-8b64-988f21c5d054";
 
         protected int MaximumSamplesCount => 10;
 
         protected int MaxPageSizeSamples => 1;
 
         protected int SkipSamples => 1;
 
-        protected DateTimeOffset SamplingStartTime => DateTimeOffset.Parse("2020-10-01T00:00:00Z");
+        protected DateTimeOffset SamplingStartTime => DateTimeOffset.Parse("2021-10-01T00:00:00Z");
```

Comment:
just curious, but is this impactful or just a cosmetic change?

Reply:
The `SamplingStartTime` and `SamplingEndTime` define the time interval from which we extract data from the MA resource in some tests. The new resource we're migrating to has been created way after the time in these timestamps, so there's no data to get in these intervals.